### PR TITLE
lt.po thanks adga + fr.po + all macedonia

### PR DESCRIPTION
**Pull request by pr2git**

### CODE REVIEW

File: po/el.po [hunk 1]
```diff
@@ -5642,6 +5639,9 @@ msgstr "Κανονική λειτουργία"
 msgid "North"
 msgstr "Βόρεια"
 
+msgid "North Macedonia (The Republic of)"
+msgstr "Δημοκρατία των Σκοπίων"
+
```
Comment (nautilus7) [inline po/el.po:5644]: I don't know where you got this, but they should be called "Βόρεια Μακεδονία" in Greek, since 2020.

**Comment by pr2git** [comment]
This PR create this:
https://github.com/OpenPLi/enigma2/pull/3054

**Comment by pr2git** [comment]
The problem is that the generation of the po file doesn't tag it as to be reviewed (and I don't know why).
I saw this PR so I change the translation in French but other translators won't see it clearly.

**Comment by nautilus7** [comment]
that's because the new string was added by hand in the place of the old one in each po file. That was done in https://github.com/OpenPLi/enigma2/commit/b13e51bdfab5f441b7bd25cb6d22dd4114a63b52

**Comment by WanWizard** [comment]
Sorry about that, I now get what you meant. I need a holiday... :tired_face:

**Comment by nautilus7** [comment]
We all need.

**Comment by pr2git** [comment]
I have create a PR that clean up all the translation for North Macedonia:

https://github.com/OpenPLi/enigma2/pull/3056

File: po/el.po (hunk 1 above)
Comment (WanWizard) [reply po/el.po:5644]: Committed translations for all supported languages. Issue should be fixed now.

**Pull request merged**